tests/cases/fourslash/completionEntryForUnionMethod.ts

```diff
@@ -3,7 +3,7 @@
 ////var y: Array<string>|Array<number>;
 ////y.map/**/(
 
-const text = "(method) Array<T>.map<unknown>(callbackfn: ((value: string, index: number, array: string[]) => unknown) & ((value: number, index: number, array: number[]) => unknown), thisArg: any): unknown[]";
+const text = "(method) Array<T>.map<unknown>(callbackfn: ((value: string, index: number, array: string[]) => unknown) & ((value: number, index: number, array: number[]) => unknown), thisArg?: any): unknown[]";

```

Comment:
From what I can tell - this is the correct change. It matches the display of the same method accessed on a single array (this one accesses it on a union of arrays):
```ts
var y: Array<string>
y.map(
// (method) Array<string>.map<unknown>(callbackfn: (value: string, index: number, array: string[]) => unknown, thisArg?: any): unknown[]
```